study.md

```diff
@@ -67,5 +69,6 @@ Starting with a client making a GET request to a particular URL, describe how
 the parts of Rails interact to produce and send a response.
 
 ```md
-<!-- your answer here -->
+Then the web server receives the request. The server uses routes to find which controller to use. The web server uses the dispatcher to create a new controller, call the action and pass the parameters.
+The controller is analyzing  the user request gives an order to model what to do with it. Model is a ruby class it talks to database - find, store, validate data - does the heavy lifting. And at the end of this the model finds the right information and send it to view to wrap it in the format to display then this format gets back to the server and it sends to the browser for a user display. 
```

Comment:
The model doesn't send anything to a view.  The controller is the one that renders responses to the client.